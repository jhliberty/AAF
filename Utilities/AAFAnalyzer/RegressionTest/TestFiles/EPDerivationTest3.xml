<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">
<aaf-file>
<!-- The following test cases should not cause failures because they are not composition Mobs.-->
 <clip name="Initial->Clip">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <eoc length="0" track-type="none"/>
  </timeline-mob-slot>
 </clip>
 <file-source sample-rate-numerator="1" sample-rate-denominator="1" name="Initial->File Source">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <eoc length="0" track-type="none"/>
  </timeline-mob-slot>
 </file-source>
 <recording-source name="Initial->Recording Source">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <eoc length="0" track-type="none"/>
  </timeline-mob-slot>
 </recording-source>
 <import-source name="Initial->Import Source">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <eoc length="0" track-type="none"/>
  </timeline-mob-slot>
 </import-source>
 <tape-source name="Initial->Tape Source">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <eoc length="0" track-type="none"/>
  </timeline-mob-slot>
 </tape-source>
 <film-source name="Initial-Film Source">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <eoc length="0" track-type="none"/>
  </timeline-mob-slot>
 </film-source>
</aaf-file>
