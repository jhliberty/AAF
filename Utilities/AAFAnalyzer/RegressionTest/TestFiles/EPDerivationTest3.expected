        Name:   Mob Derivation Test
        Desc:   Verify the correctness of mob derivation chains.
        Result: Passed
        Reason: 
        Covered Requirements: REQ_EP_017; REQ_EP_026; REQ_EP_031; REQ_EP_036; REQ_EP_045; REQ_EP_050; REQ_EP_055; REQ_EP_063; REQ_EP_072; REQ_EP_076; REQ_EP_083; REQ_EP_258; REQ_EP_259; REQ_EP_260; REQ_EP_261; REQ_EP_262
        Noted Requirements: None
        Passing Requirements: REQ_EP_025.2
        Info Requirements: None
        Warning Requirements: None
        Failing Requirements: None

            Name:   Top-level Composition Unreferenced
            Desc:   A top-level composition shall not be referenced by another Mob in the AAF file.
            Result: Passed
            Reason: No top level CompositionMob objects are referenced by other Mob objects in the file.
            Covered Requirements: None
            Noted Requirements: None
            Passing Requirements: REQ_EP_025.2
            Info Requirements: None
            Warning Requirements: None
            Failing Requirements: None

